config/prod/prow/core/plugins.yaml

```diff
@@ -80,6 +80,9 @@ config_updater:
       name: config
     config/prod/prow/core/plugins.yaml:
       name: plugins
+    label_sync/labels.yaml:
```

Comment:
Why not let the prow job use what's in master branch directly?

Reply:
You are right, I don't see any reason that needs an extra config map (I copied this setting from k8s test-infra).
Changed the job to run on master branch directly as a postsubmit job.